src/app/collective-rewards/rewards/builders/hooks/useClaimBuilderRewards.ts

```diff
@@ -3,9 +3,15 @@ import { GaugeAbi } from '@/lib/abis/v2/GaugeAbi'
 import { Address } from 'viem'
 import { useWaitForTransactionReceipt, useWriteContract } from 'wagmi'
 import { useGetBuilderRewards } from '@/app/collective-rewards/rewards'
+import { useBuilderContext } from '../../../user'
 
-const useClaimBuilderReward = (gauge: Address, rewardToken?: Address) => {
+const useClaimBuilderReward = (builder: Address, gauge: Address, rewardToken?: Address) => {
   const { writeContractAsync, error: executionError, data: hash, isPending } = useWriteContract()
+  const { getBuilderByAddress } = useBuilderContext()
+
+  const claimingBuilder = getBuilderByAddress(builder)
+  console.log('🚀 ~ useClaimBuilderReward ~ claimingBuilder:', claimingBuilder)
+  const isPaused = !!claimingBuilder?.stateFlags?.paused
```

Comment:
is it ok to show v1 (no flags) as paused?

Reply:
not really, let me update that, but at this point I dont think that they could see this.